specification/apimanagement/resource-manager/Microsoft.ApiManagement/stable/2019-12-01/definitions.json

```diff
@@ -5525,6 +5525,18 @@
         "negotiateClientCertificate": {
           "type": "boolean",
           "description": "Determines whether gateway requests client certificate"
+        },
```

Comment:
Adding these within an API version can cause undesired effects as customers use mixed SDK versions where only latest can deserialize these properties. Additionally, Azure Policy becomes unpredictable if properties customers use in policy evaluations are added within an API version. Can they be added in a new version?